webapp/src/js/components/Plot/Table/Actions.js

```diff
@@ -87,7 +90,14 @@ let TablePlotActions = React.createClass({
       label: table.capNamePlural
     }));
 
-
+    this.definedQuery = query;
```

Comment:
Funny, I did actually do something like that and then scrapped it!
...reassessing...